### DISCUSSION TITLE
Paid domains / Username

### DISCUSSION THREAD

**Discussion by vertigo220** [anonaddy/anonaddy#410] [Q&A]
Just tried signing up but my preferred username is taken already. Considering I don't want to use either of the free domains, I'd like to know a) what the paid domain options are, b) if I could use my preferred username with one of those assuming it's not taken there as well (or if a username is only allowed once for _all_ domains, which would be unfortunate), and c) if the answer to b is yes, how I would go about signing up, since I can't use the username as it's taken.

**Comment by willbrowningme** [comment]
Once a username is taken it can be used for all domains on the service.

So if you chose `johndoe` as your username you would be able to use this with any of the available domains (paid and free) e.g *@johndoe.anonaddy.com etc.

I don't publicly talk about the paid domains but if you sign up and are not happy with them I will happily offer you a full refund.

**Comment by willquill** [comment]
@vertigo220 If you are still considering using anonaddy (addy.io now), it is highly recommended to use the following strategy:

- Purchase your own domain, like vertigo220.com.
- Have vertigo220.com MX records point to your primary mail host (mailbox.org, protonmail, tutanota, or literally any other mail server).
- Have mail.vertigo220.com MX records point to addy.io. This becomes the domain you use for personally identifiable things. I use mail.myowndomain.com for things like Amazon, eBay, Steam, etc.
- For "throwaway" accounts where you don't want to have any association with yourself, like a random Reddit account to stay anonymous, just use one of the addy.io domains.
- Add `me@vertigo220.com` or whatever you use as a receipient in addy.io so all mail gets forwarded there.

Why do it like this:

- This way, if you discover you want to use a different aliasing service, or stop using an alias service at all, you can point your mail.vertigo220.com MX records to your mail server (or aliasing server) of choice. You aren't locked into addy.io. You can take your addresses with you.
- For your throwaway emails, like the Reddit example above, if you want to leave addy.io, you'd have to change your email before leaving addy.io or just deal with the loss of access to that email.

Personally, I think there are very few reasons to use username domains. Like if you have the username `vertigo220` in addy.io, you can create emails like `whatever.alias342@vertigo220.anonaddy.com`. But that email is both locked into using the addy.io service _and_ personally identifiable.

The only one I can think of is if you don't have access to a computer/phone and want to create an alias on the fly that meets the criteria of *not using your own domain* and *is relatively anonymous*, you could have a random username in addy.io like `kw4pd1`, enable catch-all, and then know that if you are giving it out to someone, like a stranger or a random sign-up sheet somewhere, you will receive emails at `anyprefix@kw4pd1.anonaddy.com`. The result is an email that has nothing to do with your vertigo220 identity and doesn't require a computer to create (since you have memorized your username and enabled catchall). The downside is you can't take it with you if you leave the service.
